EndlessClient/Controllers/ArrowKeyController.cs

```diff
@@ -100,8 +104,11 @@ private bool CurrentDirectionIs(EODirection direction)
         }
 
         private void FaceOrAttemptWalk(EODirection direction)
-        {
-            if (!CurrentDirectionIs(direction) && _characterProvider.MainCharacter.RenderProperties.IsActing(CharacterActionState.Standing))
+		{
+			if (_playerInfoProvider.IsPlayerFrozen)
```

Comment:
If `IsPlayerFrozen` is checked here, it'll never get to the point of validating the action in `AttemptToStartWalking`. I don't see any action being taken on `WalkValidationResult.Frozen` - is there supposed to be a status label message?